__test__/users.test.js

```diff
@@ -37,4 +39,85 @@ describe('PATCH /:username/image/upload', () => {
         expect(res.body.message).toBe('Error user acces');
       });
   });
+  xit('should return 200 for succesful upload', async () => {
```

Comment:
@iamcristos shoul this be `xit()` instead of `it()`?

Reply:
I intentionally skipped the test when pushing up.... that was supposed to test image upload when the test is run and would always upload an image on cloudinary...

Reply:
Okay cool